e2e-test/cypress/integration/search_spec.js

```diff
@@ -1,8 +1,10 @@
 describe("Navigation", function() {
```

Comment:
use 2 different contexts to describe project search and subproject search

Reply:
done